<launch>
<node pkg="stereo_visual_odometry" type="pose_estimation_node" name="pose_estimation_node"  output="screen" /> 
<param name="calib_yaml" type="str" value="$(find stereo_visual_odometry)/calibration/morphin.yaml"/>

<!-- Run orientation estimator-->
<node pkg="imu_ekf_ros" type="init_imu_ekf_node" name="init_ekf"  output="screen"/> 
<node pkg="imu_ekf_ros" type="imu_ekf_node" name="ekf" output="screen"/>

<param name="use_vo" type="bool" value="true"/>
<param name="logging_path" type="bool" value="false"/>
</launch>
